lib/services/ios-project-service.ts

```diff
@@ -93,13 +93,23 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		return this._platformData;
 	}
 
-	public async validateOptions(projectId: string, provision: true | string): Promise<boolean> {
+	public async validateOptions(projectId: string, provision: true | string, teamId: true | string): Promise<boolean> {
+		if (provision && teamId) {
+			this.$errors.failWithoutHelp("The options --provision and --teamId are mutually exclusive.");
+		}
+
 		if (provision === true) {
-			await this.$iOSProvisionService.list(projectId);
+			await this.$iOSProvisionService.listProvisions(projectId);
 			this.$errors.failWithoutHelp("Please provide provisioning profile uuid or name with the --provision option.");
 			return false;
 		}
 
+		if (teamId === true) {
+			await this.$iOSProvisionService.listTeams();
+			this.$errors.failWithoutHelp("Please provide team id or team name with the --teamId options.");
+			return false;
```

Comment:
I made a PR for 'never' but it is completely unrelated (I think) with this current PR:
https://github.com/telerik/mobile-cli-lib/pull/1006